docs/network/evm-nodes.md

```diff
@@ -17,6 +17,27 @@ You can get the kernel by importing it from a running Etherlink Smart Rollup nod
 
 Running an Etherlink EVM node on Etherlink Mainnet requires a computer with 500GB of disk space and at least 16GB RAM.
 
+## Modes
+
+The EVM node supports these history modes:
+
+- `archive` (the default): The node stores a copy of all available Etherlink information.
+
+- `full` (available starting with version 0.17): The node stores all of the necessary information to construct the current Etherlink state.
+
+- `rolling` (available starting with version 0.16): The node stores the current context plus the complete transaction data for a certain number of previous days.
+The default `rolling` mode stores the complete data for the previous 14 days.
+
+For more information about modes, see [History modes](https://octez.tezos.com/docs/user/history_modes.html) in the Octez documentation.
+
+You can switch from the other modes to `rolling` mode or `full` mode with the command `octez-evm-node switch history to <MODE>`, where `<MODE>` is the new mode.
+You cannot switch from another mode to `archive` mode; to set up an archive node, you must start with an `archive` mode snapshot.
+
+For example, you can switch to `rolling` mode with the command `octez-evm-node switch history to rolling:<DAYS>`, where `<DAYS>` is the number of days of compete data to keep.
+If you use this command to store more days of rolling data, such as switching from `rolling:1` to `rolling:5`, the node does not immediately download the previous 5 days of data.
+Instead, it begins storing complete data, so after 5 days it has enough data to be in `rolling:5` mode.
+You can also start a node in `rolling:5` mode by downloading a rolling snapshot that is 5 days old or by switching from `archive` mode or `full` mode.
```

Comment:
You can indeed. But, from observation that needs to be confirm, the data directory will be larger if switching from archive to rolling compared to re-using one of our rolling snapshot. I am not sure why, potentially due to Irmin internals.

Reply:
“may increase the size of the data directory, compared to running the node with the new mode from genesis” would be more precise, but I think your version is probably good enough for readers; maybe entering too much into the details here is more harmful than helpful.

Reply:
Changed to "For example, switching from another mode to `rolling` mode requires more space than starting a new node in `rolling` mode from a new data directory with the same retention period."